client/state/timezones/reducer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS
+} from 'state/action-types';
+import { timezonesSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+/**
+ * Collect and store timezones data
+ *
+ * @param  {Object} state - Current state
+ * @param  {Object} action - action payload
+ * @return {Object} updated state
+ */
+export const items = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, {
+		manual_utc_offsets,
+		timezones_by_continent,
+	} ) => ( { manual_utc_offsets, timezones_by_continent } ),
+}, timezonesSchema );
+
+/**
+ * Track the timezones requesting process
+ *
+ * @param  {Object} state - current state
+ * @param  {Object} action - action payload
+ * @return {Object} updated state
+ */
+export const requesting = createReducer( false, {
+	[ TIMEZONES_REQUEST ]: () => true,
+	[ TIMEZONES_REQUEST_FAILURE ]: () => false,
+	[ TIMEZONES_REQUEST_SUCCESS ]: () => false,
+} );
```

Comment:
For simple reducers like this one we may want to consider writing them as simpler functions. These run on every action and adding the extra overhead from `createReducer()` is just one more thing to run hundreds of times over and over.